### PR TITLE
Cloud changelog - Aug 11th

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -3,12 +3,58 @@
...
 - [Release 2021-06-16](#release-2021-06-16)
 - [Release 2021-06-02](#release-2021-06-02)
-- [Release 2021-05-21](#release-2021-05-21)
+
+## Release 2021-08-11
+
+### Highlights
+
+#### Added a global product launcher
+ -
```
Comment (amyblais) [inline source/install/cloud-changelog.md:18]: Left this blank for now as this setting may be left as disabled.

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -3,12 +3,58 @@
...
+
+#### Beta features promoted from Beta to General Availability
+   - Archived channels
+   - Compliance exports
+   - Custom terms of service
+   - Guest Accounts
+   - System Roles
+   - Plugins
+
+#### Focalboard
```
Comment (amyblais) [inline source/install/cloud-changelog.md:28]: Chen, now that Focalboard is released to Cloud, do we want to mention it in the Cloud changelog?

**Comment by amyblais** [comment]
@agnivade @streamer45 I left out v6.0 database / backend changes from this public Cloud changelog because they are likely less relevant for Cloud customers. I will document them all in the v6.0 self-managed changelog. I've listed them all in this doc if you want to take a look and let me know if there is anything that should be noted for Cloud customers https://docs.google.com/document/d/1Qog2pgc8-bypvYZweh41TqRN_XD2EGcS7U6Ej_xlY_4/edit#heading=h.d9ewlua13epk.

**Review by agnivade — APPROVED** [review]

File: source/install/cloud-changelog.md (hunk 2 above)
Comment (amyblais) [reply source/install/cloud-changelog.md:28]: cc @chenilim

File: source/install/cloud-changelog.md (hunk 1 above)
Comment (esethna) [reply source/install/cloud-changelog.md:18]: @amyblais we should be keeping this feature flag disabled until Aug 16th I imagine

**Review by esethna — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Review by wiersgallak — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]
Thanks!

**Review by streamer45 — APPROVED** [review]

File: source/install/cloud-changelog.md (hunk 2 above)
Comment (chenilim) [reply source/install/cloud-changelog.md:25]: I think this note is fine for now. We are currently in the middle of the rollout, and the plan of record is to roll out to 100% of cloud instances by Aug 16.

**Review by chenilim — APPROVED** [review]

**Pull request merged**